src/main/resources/templates/index.html

```diff
@@ -1,15 +1,34 @@
 <!DOCTYPE html>
-<html xmls:th="www.thymeleaf.org">
+<html xmlns:th="www.thymeleaf.org">
 <head>
 <meta charset="ISO-8859-1">
 <title>Hello</title>
 </head>
 <body>
-<img src="kanaAirlines.png" alt="Kana Airlines">
-<h2>Hello World!</h2>
+    <img src="kanaAirlines.png" alt="Kana Airlines">
+    <h2>Book a flight!</h2>
 
+    <form action="#" th:action="@{/flight/reservation}" method="post">
+        <h3>Select your departing city</h3> <br />
+<!--        <select th:name="departureAirport" id="departureAirport" required>-->

```

Comment:
We will definitely switch over to drop down menus with sprint 2.

Reply:
Drop downs added in my pull request